hub/integrations/pytorch.py

```diff
@@ -57,19 +62,41 @@ def _read_and_store_chunk(
     return chunk_size
 
 
-def dataset_to_pytorch(dataset, transform: Callable = None, workers: int = 1):
+def dataset_to_pytorch(
+    dataset,
+    transform: Optional[Callable] = None,
+    workers: int = 1,
+    tensors: Optional[List] = None,
```

Comment:
```suggestion
    transform: Callable = None,
    workers: int = 1,
    tensors: Sequence = None,
```

nit-pick: use `Sequence` instead of `List` whenever possible. Also, `Optional` is not required when explicitly doing `= None`

Reply:
Regarding `Optional`: https://stackoverflow.com/questions/62732402/can-i-omit-optional-if-i-set-default-to-none